docs/user/dk_bench.md

```diff
@@ -0,0 +1,185 @@
+# DK-Bench (Domain-Knowledge Bench)
+
```

Comment:
Seems it would be better to add a short introduction in `README.md` and link to user doc.